.github/CODEOWNERS

```diff
@@ -308,7 +308,7 @@ platform/security/policy/security-policy-api/src/main/java/org/codice/ddf/securi
 # SPATIAL
 
 catalog/spatial/kml/spatial-kml-transformer/src/main/java/org/codice/ddf/spatial/kml/transformer/KmlStyleMapEntry.java @codice/whitelist
-catalog/spatial/kml/spatial-kml-transformer/src/main/java/org/codice/ddf/spatial/kml/transformer/KMLTransformer.java @codice/whitelist
+catalog/spatial/kml/spatial-kml-transformer/src/main/java/org/codice/ddf/spatial/kml/transformer/KmlTransformer.java @codice/whitelist
```

Comment:
Ah this is a whitelisted class? let me double check to make sure we are allowed to change the name. 

also I wonder why the group was not tagged when the file was changed (it should have auto requested review from @codice/whitelist) 😢

Reply:
❗️ So since this is listed under a whitelisted package we cannot change the name.

Reply:
Fixed, good catch